### PR TITLE
Add known limitations section

### PR DESCRIPTION
**Pull request by e-grapp**
# Description
Added a section to include known issues and limitations 

# Reasons
https://circleci.atlassian.net/browse/SC-736

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/status-badges.adoc [hunk 1]
```diff
@@ -161,3 +161,8 @@ image::{{site.baseurl}}/assets/img/docs/svg-failed.png[Failed `svg` version]
 image::{{site.baseurl}}/assets/img/docs/shield-passing.png[Passing shield version]
 
 image::{{site.baseurl}}/assets/img/docs/shield-failing.png[Failing shield version]
+
+[#known-issues-and-limitations]
+== Known issues and limitations
+
+* In certain cases where there are multiple workflows running within a pipeline, there is a known bug where the status badge will show the result (Red or Green) of the latest workflow that has completed running instead of waiting for all of the workflows in a pipeline to finish terminating and then updating the badge (Red or Green) to use the result of the last workflow.
```
Comment (rosieyohannan) [inline jekyll/_cci2/status-badges.adoc:168]:
```suggestion
* In cases where there are multiple workflows running within a pipeline, the status badge will show the result (red or green) of the _latest_ workflow that has finished running. This is a bug. The expected behavior is for the badge to update to show the result of the _last_ workflow once the pipeline is complete.
```

**Comment by sebastian-lerner** [comment]
@rosieyohannan any chance you can help merge this?

**Comment by imlogang** [comment]
I fixed the `status-badge.adoc` lint error but seems the build job failed, so will need @rosieyohannan to rerun.

**Review by imlogang — APPROVED** [review]

**Comment by imlogang** [comment]
SSO was being weird. 
Was able to rerun the build job.

**Pull request merged**